formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -39,13 +41,14 @@ public sealed class Cbor(
     internal val writeValueTags: Boolean,
     internal val verifyKeyTags: Boolean,
     internal val verifyValueTags: Boolean,
+    internal val preferSerialLabelsOverNames: Boolean,
     override val serializersModule: SerializersModule
 ) : BinaryFormat {
 
     /**
      * The default instance of [Cbor]
      */
-    public companion object Default : Cbor(false, false, true, true, true, true, EmptySerializersModule())
+    public companion object Default : Cbor(false, false, true, true, true, true, true, EmptySerializersModule())
```

Comment:
as in: if label is present, it will be preferred by default?

Reply:
I'd be more verbose in documenting this

Reply:
Is it better now?